playbooks/ansible-core-ci/pre.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: localhost
```

Comment:
I think this should be 'all', if you use localhost here, that is the zuul-executor.